cli/src/lib/libDefs.js

```diff
@@ -159,10 +159,24 @@ async function addLibDefs(pkgDirPath, libDefs: Array<LibDef>, validationErrs?: V
 /**
  * Given a 'definitions/npm' dir, return a list of LibDefs that it contains.
  */
+ const basePathRegex = /definitions\/npm\/[^\/]*\/?/;
```

Comment:
This needs to consider Windows, too. Can you use the `RegExp` constructor + `path.sep`?